### PR TITLE
ENT-3832: Improve error logging

### PR DESCRIPTION
**Pull request by Januson**
- Make sure exceptions are propagated/logged so that stacktraces are not lost
- Merge identical catch blocks

### CODE REVIEW

File: server/src/main/java/org/candlepin/model/CdnCurator.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public Cdn getByLabel(String label) {
                 .uniqueResult();
         }
         catch (NonUniqueResultException e) {
-            throw new IllegalStateException("Multiple CDN instances found with the same label: " + label);
+            throw new IllegalStateException("Multiple CDN instances found with the same label: " + label, e);
```
Comment (Ceiu) [inline server/src/main/java/org/candlepin/model/CdnCurator.java:49]: Is this going to provide any additional details that will help debug it? I feel like it'll just add another chunk to the stack trace that we'll ignore.

**Review by Ceiu — CHANGES_REQUESTED** [review]
Couple requests here. I'm not sure how I feel about renaming all exceptions (or any variable for that matter) to 'e'. I get that it's common practice, it just makes things a tad obnoxious when using multiselect to do fast local refactors.

File: common/src/main/java/org/candlepin/common/util/VersionUtil.java [hunk 2]
```diff
@@ -54,17 +54,17 @@ private VersionUtil() {
                 map.put("release", props.getProperty("release"));
             }
         }
-        catch (IOException ioe) {
-            log.error("Can not load version.properties", ioe);
+        catch (IOException e) {
+            log.error("Can not load version.properties", e);
         }
         finally {
```
[the remark this answers is not in the document]
Comment (Januson) [reply common/src/main/java/org/candlepin/common/util/VersionUtil.java]: Done.

File: server/src/main/java/org/candlepin/audit/EventFilter.java [hunk 3]
```diff
@@ -109,9 +109,10 @@ private void fillEventTypeAndTargetFromConfig(Set<EventTypeAndTarget> set, List<
                 Target target = Enum.valueOf(Target.class, split[1].trim());
                 set.add(new EventTypeAndTarget(type, target));
             }
-            catch (Exception ex) {
+            catch (Exception e) {
                 throw new IllegalArgumentException("Invalid filtering tuple: " + item +
-                        " . Please use only enum values of Type and Target enums, e.g. MODIFIED-CONSUMER");
+                        " . Please use only enum values of Type and Target enums, e.g. MODIFIED-CONSUMER",
```
[the remark this answers is not in the document]
Comment (Januson) [reply server/src/main/java/org/candlepin/audit/EventFilter.java]: Done.

File: server/src/main/java/org/candlepin/model/CdnCurator.java (hunk 1 above)
Comment (Januson) [reply server/src/main/java/org/candlepin/model/CdnCurator.java:49]: I just added it everywhere where it missing. I agree that in this specific case it's not adding a lot of value, but I think it's generally better to just have them.

**Review by Ceiu — APPROVED** [review]

**Pull request merged**